src/components/select/select.component.ts

```diff
@@ -46,8 +46,8 @@ export class SelectComponent<T> implements OnInit, OnChanges, OnDestroy, Control
 
     /** The text in the input area. This is used to filter the options dropdown. */
     @Input()
-    set input(value: string) {
-        this._input$.next(value);
+    set input(value: InputValue) {

```

Comment:
I don't think we can change the type of `input` and `inputChange`, this would introduce a breaking change for anyone relying on these